code/modules/clothing/clothing.dm

```diff
@@ -700,6 +700,16 @@ BLIND     // can't see anything
 				to_chat(usr, "<span class='notice'>Your suit will now only report your exact vital lifesigns.</span>")
 			if(3)
 				to_chat(usr, "<span class='notice'>Your suit will now report your exact vital lifesigns as well as your coordinate position.</span>")
+	else if(istype(loc, /mob))
+		switch(sensor_mode)
+			if(0)
+				usr.visible_message("<span class='warning'>[usr] disables [loc]'s remote sensing equipment.</span>")
+			if(1)
+				usr.visible_message("[usr] turns [loc]'s remote sensors to binary.")
+			if(2)
+				usr.visible_message("[usr] sets [loc]'s sensors to track vitals.")
+			if(3)
+				usr.visible_message("[usr] sets [loc]'s sensors to maximum.")

```

Comment:
All of the ones that still don't have a span should also be a `span class='notice'` pls.

Sorry for not having mentioned that earlier.